deploy/helm/metacontroller/values.yaml

```diff
@@ -64,3 +64,9 @@ cacheFlushInterval: 30m
 
 # The name of the PriorityClass that will be assigned to metacontroller
 priorityClassName: ""
+
```

Comment:
I am thinking we can default this as:
```
podDisruptionBudget: {}
```

Reply:
Thats also an option, I like having defaults in the chart.